src/config.ts

```diff
@@ -176,6 +176,24 @@ export interface ValidationSchemaPluginConfig extends TypeScriptPluginConfig {
    * ```
    */
   scalarSchemas?: ScalarSchemas
+  /**
+   * @description Fallback scalar type for undefined scalar types in the schema not found in `scalarSchemas`.
+   *
+   * @exampleMarkdown
+   * ```yml
+   * config:
+   *   schema: yup
+   *   defaultScalarSchema: yup.unknown()
+   * ```
+   *
+   * @exampleMarkdown
+   * ```yml
+   * config:
+   *   schema: zod
+   *   defaultScalarSchema: z.unknown()
+   * ```
+   */
+  defaultScalarTypeSchema?: string
```

Comment:
@jocisneros Could you add to also README?

Reply:
done 🫡